keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ArtifactRepositoryTests.kt

```diff
@@ -167,10 +179,26 @@ abstract class ArtifactRepositoryTests<T : ArtifactRepository> : JUnit5Minutests
               .isFalse()
           }
         }
+
+        test("the artifact version is pending in the environment") {
+          val environment1Versions = subject
+            .versionsByEnvironment(manifest)
+            .first { it.name == environment1.name }
+            .artifacts
+            .first { it.name == artifact1.name && it.type == artifact1.type }
+            .versions
```

Comment:
These lines repeat across all new tests... Would be nice to extract them into a function, maybe?

Reply:
I made it an extension on the fixture